chains/v20/chains.json

```diff
@@ -4562,8 +4562,8 @@
         ],
         "nodes": [
             {
-                "url": "wss://fullnode.parachain.centrifuge.io",
```

Comment:
that was a node for Centrifuge EVM

Reply:
I would prefer not remove this node from chains.json as extracting EVM chains from Astar, Centrifuge and other is in development stage atm